afs/afs-ws/afs-ws-client-utils/src/main/java/com/powsybl/afs/ws/client/utils/RemoteServiceConfig.java

```diff
@@ -27,6 +27,14 @@
 
     private boolean secure;
 
+    private boolean autoreconnect;
```

Comment:
Refactor at least to autoReconnect (auto and reconnect are two different words). Maybe it should be autoReconnectionEnabled

Reply:
Ok, it is done